submarine-server/server-core/src/main/java/org/apache/submarine/server/workbench/database/entity/Job.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.submarine.server.workbench.database.entity;
+
+public class Job extends BaseEntity {
+
+  // (Todo) Add dict configuration.
```

Comment:
```
// (Todo) Add dict configuration.
```
Is this an invalid comment, or will this feature need to be added later?
If you need to add this function in the future, you need to change it to `TODO (you name) Add dict configuration.`

Reply:
Thanks @liuxunorg, Done.